tools/clang/unittests/HLSLExec/LongVectorOps.def

```diff
@@ -129,28 +134,58 @@ OP_DEFAULT(BinaryComparison, NotEqual, 2, "", "!=")
 
 OP_DEFAULT(Binary, Logical_And, 2, "and", ",")
 OP_DEFAULT(Binary, Logical_Or, 2, "or", ",")
-OP(Ternary, Select, 3, "TestSelect", "", " -DFUNC_TEST_SELECT=1", SelectCond, Default2, Default3)
+OP(Ternary, Select, 3, "TestSelect", "", " -DFUNC_TEST_SELECT=1",
+   "LongVectorOp", SelectCond, Default2, Default3)
 
 // For the 'any' and 'all' reduction operation, we test with three different
 // input data sets to cover the different cases of the operation. 'Any' returns
 // true if any of the inputs are non-zero, false otherwise. This is in
 // contrast to other ops where we can get coverage simply with different values
 // across the input set.
-OP(Reduction, Any_Mixed, 1, "any", "", " -DIS_REDUCTION_OP=1", Default1, 
-   Default2, Default3)
-OP(Reduction, Any_Zero, 1, "any", "", " -DIS_REDUCTION_OP=1", Zero,
-   Default2, Default3)
-OP(Reduction, Any_NoZero, 1, "any", "", " -DIS_REDUCTION_OP=1", NoZero,
-   Default2, Default3)
-OP(Reduction, All_Mixed, 1, "all", "", " -DIS_REDUCTION_OP=1", Default1, 
-   Default2, Default3)
-OP(Reduction, All_Zero, 1, "all", "", " -DIS_REDUCTION_OP=1", Zero,
-   Default2, Default3)
-OP(Reduction, All_NoZero, 1, "all", "", " -DIS_REDUCTION_OP=1", NoZero,
-   Default2, Default3)
-
+OP(Reduction, Any_Mixed, 1, "any", "", " -DIS_REDUCTION_OP=1", "LongVectorOp",
+   Default1, Default2, Default3)
+OP(Reduction, Any_Zero, 1, "any", "", " -DIS_REDUCTION_OP=1", "LongVectorOp",
+   Zero, Default2, Default3)
+OP(Reduction, Any_NoZero, 1, "any", "", " -DIS_REDUCTION_OP=1", "LongVectorOp",
+   NoZero, Default2, Default3)
+OP(Reduction, All_Mixed, 1, "all", "", " -DIS_REDUCTION_OP=1", "LongVectorOp",
+   Default1, Default2, Default3)
+OP(Reduction, All_Zero, 1, "all", "", " -DIS_REDUCTION_OP=1", "LongVectorOp",
+   Zero, Default2, Default3)
+OP(Reduction, All_NoZero, 1, "all", "", " -DIS_REDUCTION_OP=1", "LongVectorOp",
+   NoZero, Default2, Default3)
 OP_DEFAULT(Reduction, Dot, 2, "dot", ",")
 
 OP_DEFAULT_DEFINES(Unary, ShuffleVector, 1, "TestShuffleVector", "", " -DFUNC_SHUFFLE_VECTOR=1")
 
+
+#define OP_LOAD_AND_STORE(SYMBOL, SHADER_NAME)                                 \
+   OP(UNARY, SYMBOL, 1, "", "", "", "LongVectorOp_" SHADER_NAME,               \
+       Default1, Default2, Default3)
+#define OP_LOAD_AND_STORE_SB(SYMBOL, SHADER_NAME)                              \
+   OP(UNARY, SYMBOL, 1, "", "", " -DUSE_STRUCTURED_BUFFER=1",                  \
+      "LongVectorOp_" SHADER_NAME, Default1, Default2, Default3)
+
+// BAB == Byte Address Buffer
```

Comment:
Really wasn't sure about naming on these. I hated the even longer verbose names. These don't feel great either. But I dislike them less.